exporter/sumologicexporter/config.go

```diff
@@ -104,9 +113,9 @@ const (
 	// DefaultMaxRequestBodySize defines default MaxRequestBodySize in bytes
 	DefaultMaxRequestBodySize int = 20_971_520
```

Comment:
Nitpick ( I know this has not changed in this PR): can we change this to something like `20 * 1024 *1024`?